src/components/List.js

```diff
@@ -55,6 +94,7 @@ const List = ({ token }) => {
                 id={doc.id}
                 onChange={handleCheckbox}
                 checked={checkPurchasedDate(doc.lastPurchased)}
+                disabled={checkPurchasedDate(doc.lastPurchased)}
```

Comment:
@LINDGRENBA and I think `checked` and `disabled` follow the same logic so passing the `checkedPurchaseDate` function to `disabled` is the route we took. Once checked the checkbox is disabled so a user can't continually tap the checkbox and update the `lastPurchased` date